x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -291,6 +292,24 @@ export const DataGrid: FC<Props> = memo(
             <EuiSpacer size="m" />
           </div>
         )}
+        {ccsWarning && (
+          <div data-test-subj={`${dataTestSubj} ccsWarning`}>
+            <EuiCallOut
+              title={i18n.translate('xpack.ml.dataGrid.CcsWarningCalloutTitle', {
+                defaultMessage: 'Cross Cluster Search returned no fields data.',
```

Comment:
From the current docs, looks like `Cross-cluster search` is the usual formatting for this term.

Reply:
Fixed in fc4e950.